pages/about.tsx

```diff
@@ -65,7 +65,7 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
         since 2018 it has been run by DDD WA Inc.
       </SafeLink>
       . We are now in our fifth year, which we think is worth celebrating!{' '}
```

Comment:
Do we need to change this text `We are now in our fifth year`?

Reply:
Yeah we can probably remove this sentence.

Reply:
Cool, have removed it.